desktop/app/menu-bar.js

```diff
@@ -3,21 +3,29 @@ import {resolveImage, resolveRootAsURL} from '../resolve-root'
 import hotPath from '../hot-path'
 import menubar from 'menubar'
 
-let color = 'white'
-let platform = ''
-
-if (process.platform === 'darwin') {
-  color = (systemPreferences && systemPreferences.isDarkMode()) ? 'white' : 'black'
-} else if (process.platform === 'win32') {
-  color = 'black'
-  platform = 'windows-'
-}
+let iconType: 'regular' | 'update' | 'badged' = 'regular'
+
+const getIcon = () => { // eslint-disable-line arrow-parens
+  const devMode = __DEV__ ? '-dev' : ''
+  let color = 'white'
+  let platform = ''
 
-const devMode = __DEV__ ? '-dev' : ''
+  if (process.platform === 'darwin') {
+    color = (systemPreferences && systemPreferences.isDarkMode()) ? 'white' : 'black'
+  } else if (process.platform === 'win32') {
+    color = 'black'
+    platform = 'windows-'
+  }
 
-const icon = resolveImage('menubarIcon', `icon-${platform}keybase-dog-regular-${color}-22${devMode}@2x.png`)
-const loadingIcon = resolveImage('menubarIcon', `icon-${platform}keybase-dog-update-${color}-22${devMode}@2x.png`)
-const badgedIcon = resolveImage('menubarIcon', `icon-${platform}keybase-dog-badged-${color}-22${devMode}@2x.png`)
+  switch (iconType) {
+    case 'regular':
+      return resolveImage('menubarIcon', `icon-${platform}keybase-dog-regular-${color}-22${devMode}@2x.png`)
+    case 'update':
+      return resolveImage('menubarIcon', `icon-${platform}keybase-dog-update-${color}-22${devMode}@2x.png`)
+    case 'badged':
+      return resolveImage('menubarIcon', `icon-${platform}keybase-dog-badged-${color}-22${devMode}@2x.png`)
```

Comment:
Any reason to not string interpolate the `${iconType}` here, since all that's changing is the word?